oauth2client/transport/__init__.py

```diff
@@ -64,7 +64,27 @@ def make_authorized_http(
         http_object, credentials, refresh_status_codes=refresh_status_code)
 
 
-def request(http_object, uri, method='GET', body=None, headers=None, **kwargs):
-    """Makes an HTTP request."""
-    return get_default_transport().request(
+def _request(http_object, uri, method='GET', body=None, headers=None,
+             **kwargs):
+    """Makes an HTTP request.
+
+    Args:
+        http_object: The transport-specific http object to be used to make
+            requests.
+        uri: string, The URI to be requested.
+        method: string, The HTTP method to use for the request. Defaults
+            to 'GET'.
+        body: string, The payload / body in HTTP request. By default
+            there is no payload.
+        headers: dict, Key-value pairs of request headers. By default
+            there are no headers.
+
+    Returns:
+        An response object that will contain at least the following properties:
+
+        * status: int, the HTTP status code.
+        * headers: dict, the HTTP response headers.
+        * data: bytes, the HTTP response body.
+    """
+    return get_default_transport()._request(
```

Comment:
I know Pylint would kill you here, using a private method of an object outside of one of that objects methods.